core-rust/mesh-api-server/README.md

```diff
@@ -0,0 +1,190 @@
+- [Mesh API CLI Test Tool](https://github.com/coinbase/mesh-cli)
+
+## Supported Features
+
+| Feature                      | Status                                                                |
+| ---------------------------- | ----------------------------------------------------------------------|
+| Data API                     | Feature-complete, with some quirks                                    |
+| - `/network/list`            | Complete                                                              |
+| - `/network/status`          | Complete                                                              |
+| - `/network/options`         | Complete                                                              |
+| - `/block`                   | Feature-complete (exposes only balance-changing operations)           |
+| - `/block/transaction`       | Feature-complete (exposes only balance-changing operations)           |
+| - `/account/balance`         | Complete (historical balances available if explicitly enabled)        |
+| - `/mempool`                 | Complete (transactions are not held for a meaningful duration)        |
+| - `/mempool/transaction`     | Complete (basic operations supported)                                 |
+| Construction API             | Complete                                                              |
+| - `/construction/derive`     | Complete                                                              |
+| - `/construction/preprocess` | Complete                                                              |
+| - `/construction/metadata`   | Complete                                                              |
+| - `/construction/payloads`   | Complete (supports Withdraw and Deposit operations only)              |
+| - `/construction/combine`    | Complete                                                              |
+| - `/construction/parse`      | Complete (basic operations supported)                                 |
+| - `/construction/hash`       | Complete                                                              |
+| - `/construction/submit`     | Complete                                                              |
+
+## Additional Considerations
+
+### Accounts
+
+The current implementation has the following constraints:
+- **Supports only account components**: Returns block operations or balances exclusively for accounts. Other components (e.g., smart contracts, lockers) are ignored.
+- **Supports only Withdraw, Deposit, and FeePayment operations**: Minting and burning are skipped.
+
+These constraints simplify the implementation without causing Mesh CLI tests to fail. If non-account components must be supported, the following may be required:
+- Adding support for Minting and Burning operations.
+- Providing explicit support for non-account components in balance fetching (or using `dump_component_state()`, which is resource-intensive).
+- Exempting some addresses.
```

Comment:
One of the addresses that comes to my mind is `Faucet` component (in case of test networks), which `free`'s some test XRDs.
I think it would be the simplest to exempt it. I didn't want to go much into details in this comment but I can elaborate more here.

Reply:
But faucet isn't an account address...
And if we properly supported non-account addresses, they'd all reconcile across genesis anyway, so I don't think this is needed? Did you hit some issue which required you adding this?

Reply:
I was trying to support all addresses (not only accounts) and `Faucet` was one of the components, which was causing reconcilation to fail. AFAIR the other one was `Consensus Manager`.